### ISSUE TITLE
Questions about dataset

### ISSUE THREAD

**Issue by haowentang97** [NasimShafiee/Introvert-Human-trajectory-prediction-via-conditional-3d-attention#1]
Hi Nasim,
Thank you for your sharing of the codes.

I have somes question about dataset. 

Could you please explain how to generate the 'pos_data_train.db' file? It seems that the training data contains the testing set.  For example, for hotel, the training loader seems contain the data from hotel.

What's more, I didn't find the '.db' file for testing, could you please upload them?

Could you help me to handle the question?
Thank you very much!
Haowen

**Comment by haowentang97** [comment]
Or could you please explain how to generate the 'pos_data_train.db' file?
Thank you very much!

**Comment by while-lin** [comment]
Maybe this paper just devided a dataset such as ETH into train.csv and test.csv, the rest dataset(hotel, zara1,etc) are not used

**Comment by NasimShafiee** [comment]
Each dataset(zara1, zara2, uni, eth, hotel) is divided to train and test. The model is trained on the train set and tested on the test set of that specific dataset. We didn't consider transfer learning and zero shot learning across datasets.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by while-lin** [comment]
> Each dataset(zara1, zara2, uni, eth, hotel) is divided to train and test. The model is trained on the train set and tested on the test set of that specific dataset. We didn't consider transfer learning and zero shot learning across datasets. Get Outlook for iOS<https://aka.ms/o0ukef>
> […](#)
> ________________________________ From: while-lin ***@***.***> Sent: Saturday, October 29, 2022 10:54:25 AM To: NasimShafiee/Introvert-Human-trajectory-prediction-via-conditional-3d-attention ***@***.***> Cc: Subscribed ***@***.***> Subject: Re: [NasimShafiee/Introvert-Human-trajectory-prediction-via-conditional-3d-attention] Questions about dataset (Issue #1) Maybe this paper just devided a dataset such as ETH into train.csv and test.csv, the rest dataset(hotel, zara1,etc) are not used — Reply to this email directly, view it on GitHub<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FNasimShafiee%2FIntrovert-Human-trajectory-prediction-via-conditional-3d-attention%2Fissues%2F1%23issuecomment-1295918131&data=05%7C01%7Cshafiee.n%40northeastern.edu%7C18a2a50baa1a4333ea4508dab9d69e7f%7Ca8eec281aaa34daeac9b9a398b9215e7%7C0%7C0%7C638026628704485708%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ODZrmd%2BR8pR6y2MEX7bMk1vgc5JvID2xNRyiMORXH%2Bg%3D&reserved=0>, or unsubscribe<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAHSMRT3JKBV3T6TANCPRAR3WFVQFDANCNFSM5675MWEA&data=05%7C01%7Cshafiee.n%40northeastern.edu%7C18a2a50baa1a4333ea4508dab9d69e7f%7Ca8eec281aaa34daeac9b9a398b9215e7%7C0%7C0%7C638026628704485708%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=7YONP84CpHOvlNkbB7MZ89d%2BBW116HAHVttzlV6S3gw%3D&reserved=0>. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Yes, that's what I mean. My English is poor. Sry
